index.js

```diff
@@ -12,3 +14,15 @@ const pagination = document.querySelector('[data-js="pagination"]');
 const maxPage = 1;
 const page = 1;
 const searchQuery = "";
+
+async function fetchCharacters() {
+  try {
+    const response = await fetch("https://rickandmortyapi.com/api/character/");
+    const data = await response.json();
+    console.log(data.results);
```

Comment:
Please remove all `console.log()` statements before creating a pull request. :) These are useful for testing our functionality but should not be present in the `main` branch later on.